tests/post/steps/test_example_solution.py

```diff
@@ -0,0 +1,398 @@
+        response = get_deployment(k8s_appsv1_client, name, environment)
+        available_replicas = utils.get_dict_element(response, path)
+        assert available_replicas is not None
+        context['deployment_obj'] = available_replicas
+
+    utils.retry(
+        _wait_for_deployment,
+        times=10,
+        wait=5,
+        name="wait for available Deployments"
+    )
+
+
+@when(parsers.parse("we deactivate Solution '{name}'"))
+def deactivate_solution(request, host, name):
+    iso_root = request.config.getoption("--iso-root")
+    cmd = str(iso_root / "solutions.sh")
+    with host.sudo():
+        res = host.run(' '.join([
+            cmd,
+            'deactivate  --name {}'.format(name),
+        ]))
+        assert res.rc == 0, res.stdout
+
+
+@when(parsers.parse(
+    "we delete Solution '{name}' in environment '{environment}'"))
+def delete_solution(host, request, name, environment):
+    iso_root = request.config.getoption("--iso-root")
+    cmd = str(iso_root / "solutions.sh")
+    with host.sudo():
+        res = host.run(' '.join([
+            cmd,
+            'delete-solution --name {}'.format(environment),
+            '--solution {}'.format(name),
+        ]))
+        assert res.rc == 0, res.stdout
+
+
+@when(parsers.parse("we delete Solution environment '{environment}'"))
+def delete_solution_env(host, request, environment):
+    iso_root = request.config.getoption("--iso-root")
+    cmd = str(iso_root / "solutions.sh")
+    with host.sudo():
+        res = host.run(' '.join([
+            cmd,
+            'delete-env --name {}'.format(environment),
+        ]))
+        assert res.rc == 0, res.stdout
+
+
+@when(parsers.parse("we unimport Solution archive '{path}'"))
+def unimport_solution_archive(host, request, path):
+    iso_root = request.config.getoption("--iso-root")
+    cmd = str(iso_root / "solutions.sh")
+    with host.sudo():
+        res = host.run(' '.join([
+            cmd,
+            'unimport --archive',
+            '{}'.format(path),
+        ]))
+        assert res.rc == 0, res.stdout
+
+
+@when(parsers.parse("we delete the Solution Configuration file"))
+def remove_solution_config(host, request):
+    with host.sudo():
+        res = host.run(' '.join([
+            'rm -rf {}'.format(SOLUTION_CONFIGURATION_FILE),
+        ]))
+        assert res.rc == 0, res.stdout
+
+# }}}
+
+
+# Then {{{
+
+@then(parsers.parse("Solution archive '{name}' is imported correctly"))
+def is_imported_solution_archive(host, k8s_client, name):
+    # Solution is imported correctly if
+    # 1. Solution Configuration is created at /etc/metalk8s/solution.yaml
+    # 2. Solution Configuration has new archive path
+    # 3. Solutions ConfigMap has new Solution imported
+
+    with host.sudo():
+        if not host.file(SOLUTION_CONFIGURATION_FILE).exists:
+            assert False, 'No Solution Configuration file exist at {}'.format(
+                        SOLUTION_CONFIGURATION_FILE
+                    )
```

Comment:
Why not directly asserting ?

```suggestion
    assert host.file(SOLUTION_CONFIGURATION_FILE).exists, \
        'No Solution Configuration file exists at {}'.format(SOLUTION_CONFIGURATION_FILE)
```